### ISSUE TITLE
Error while reading from the Firehose: websocket: close

### ISSUE THREAD

**Issue by vasanth1806** [bosh-prometheus/prometheus-boshrelease#359]
Getting the below error in firehose vm
time="2019-11-20T00:39:23Z" level=error msg="Error while reading from the Firehose: websocket: close 1006 (abnormal closure): unexpected EOF" source="firehose_nozzle.go:121"
time="2019-11-20T09:49:03Z" level=info msg="Starting firehose_exporter (version=5.0.4, branch=master, revision=6f73b9dd4c8718f59ac0a5d1f03203d96fd5e7fe)" source="firehose_exporter.go:168"
time="2019-11-20T09:49:03Z" level=info msg="Build context (go=go1.11.5, user=root@719955eac311, date=20190202-06:10:56)" source="firehose_exporter.go:169"
time="2019-11-20T09:49:03Z" level=info msg="Listening on :9186" source="firehose_exporter.go:247"
time="2019-11-20T09:49:03Z" level=info msg="Starting Firehose Nozzle..." source="firehose_nozzle.go:60"
Please help out with what could be the issue..

**Comment by san4ever** [comment]
Also we observed below errors in cf-exporter in Prometheus VM
**"408 Request Timeout"**
**"Error while listing service bindings"**
Because of that data is not coming in Cloudfoundry -> Apps Latency
When we perform monit restart all in firehose VM, data is coming in Graphana Dashboard
![image](https://user-images.githubusercontent.com/29483634/69551354-9076d380-0fc2-11ea-9260-9766008c60b4.png)

**Comment by san4ever** [comment]
Hi @benjaminguttmann-avtq  
Can you please take a look on this. In all our foundations frequently we are losing the data in Cloud Foundry Graphana Dashboard

**Comment by ywei2017** [comment]
@san4ever Did you get the issue resolved? I have a similar issue, and I suspect the firehose is emitting too much data for the firehose-exporter to keep up. I don't know whether the release supports multiple `exporters` nodes.

**Comment by benjaminguttmann-avtq** [comment]
@ywei2017 I think as long as you configure the same `doppler.subscription-id` for the different firehose exporter instances that should work

**Comment by ywei2017** [comment]
@benjaminguttmann-avtq Do I need to split [monitor-cf.yml](https://github.com/bosh-prometheus/prometheus-boshrelease/blob/master/manifests/operators/monitor-cf.yml)  into separate cf-exporter from firehose-exporter?
* I assume I shouldn't run more than 1 instance of cf-exporter?
* Is the prometheus job wired to scrape from multiple firehose-exporter instances?

I can try it and see what I get. But figure to ask first.

Thanks
Yansheng

**Comment by ywei2017** [comment]
@benjaminguttmann-avtq I think I found answer to my owner question. I took monitor-cf.yml more prescriptively than necessary. I was able to split the firehose exporter into a separate instance group with multiple instances, and resolved my issue. I did find out the release needs to be updated, since some queries gave wrong results when multiple instances are involved. I create [PR 401(https://github.com/bosh-prometheus/prometheus-boshrelease/pull/401) to correct it.

Thanks
